telco-core/configuration/reference-crs-kube-compare/optional/other/sctp_module_mc.yaml

```diff
@@ -0,0 +1,25 @@
+# optional
+# count: 1
+apiVersion: machineconfiguration.openshift.io/v1
```

Comment:
This MachineConfig isn't in line with our documentation and yields a lot of false positives:
https://docs.openshift.com/container-platform/4.16/networking/using-sctp.html
https://docs.openshift.com/container-platform/4.14/networking/using-sctp.html

a) The ignition version that you're using is fairly old. The 4.14 and 4.16 doc use 3.2.0. With that said, it should probably be a regex accepting a minimum version, if possible
b)  The `            verification: {}` statement and line `filesystem: root` do not show up in our documentation
c) Either make the verification for contents of sctp-load.conf a regexp so that it can verify both the base64 version as well as the utf-8 version, or enforce the clear text version because that's the one that we are documenting. Otherwise I get results such as:
```
-          source: data:text/plain;charset=utf-8;base64,c2N0cA==
+          source: data:text/plain;charset=utf-8,sctp
```
d) The documentation instructs users to set "overwrite: true", so that line should be optional or enforced, right now it's complaining when the file is missing

Reply:
O.k., I see now that you are getting this from https://docs.openshift.com/container-platform/4.14/scalability_and_performance/telco_ref_design_specs/core/telco-core-ref-crs.html#networking-yaml_ran-core-ref-design-crs
That's a problem in the RDS then and should be fixed both in the RDS and here

Reply:
Agreed we should accept both forms and align with the documentation. I've opened a bug to track this.